packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.test.tsx

```diff
@@ -645,7 +645,7 @@ describe('DetailsHeader', () => {
 
     dropHintElement = component.find('#columnDropHint_1').getDOMNode();
     dropHintElementChildren = dropHintElement.children;
-    expect(dropHintElementChildren.item(0)!.getAttribute('style')).toEqual('display: none;');
+    expect(dropHintElementChildren.item(0)!.getAttribute('style')).toContain('display: none;');
```

Comment:
why this change `toContain`? It's not an exact match anymore?

Reply:
I guess not, since `style` now has `fontFamily` in it from my understanding.

Reply:
Yes that's why